.gitlab-ci.yml

```diff
@@ -308,9 +308,9 @@ before_script:
 # stage: deploy-part1
   MOVED_TO_GROUP_VARS: "true"
 
-.coreos_vault_upgrade_variables: &coreos_vault_upgrade_variables
+.coreos_vault_variables: &coreos_vault_variables
 # stage: deploy-part1
-  UPGRADE_TEST: "basic"
+  MOVED_TO_GROUP_VARS: "true"
```

Comment:
The docker version of coreos stalbe is 18.06, 
 kubeadm 1.11.3  cluster will appear version greater than 17.03.

Reply:
this Var can be removed